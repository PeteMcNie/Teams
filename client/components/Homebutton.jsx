import React from 'react'
import { Link } from 'react-router-dom'

const Homebutton = () => {
    return (
        <div className='link'>
            <Link to={'/'} className='home'>Home</Link>
        </div>
    )
}

export default Homebutton
