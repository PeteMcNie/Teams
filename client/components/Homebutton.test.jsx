import React from 'react'

import { renderWithRedux } from '../testing/utils'

import Homebutton from './Homebutton'

test('Homebutton component matches snapshot', () => {
  const { container } = renderWithRedux(<Homebutton />)
  expect(container).toMatchSnapshot()
})
