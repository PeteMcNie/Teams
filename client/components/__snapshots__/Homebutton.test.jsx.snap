// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homebutton component matches snapshot 1`] = `
<div>
  <div
    class="link"
  >
    <a
      class="home"
      href="/"
    >
      Home
    </a>
  </div>
</div>
`;
